app/views/address/addressList.scala.html

```diff
@@ -15,55 +15,64 @@
  *@
 
 @import config.FrontendAppConfig
-@import uk.gov.hmrc.play.views.html._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.{FormWithCSRF => PlayFrontendFormWithCSRF}
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 @import models.address.Address
 @import viewmodels.address.AddressListViewModel
 @import utils.ViewHelper.titleWithErrors
 
 @this(
-    main_template: main_template,
-    formHelper: FormWithCSRF
+    layout: templates.Layout,
+    formHelper: PlayFrontendFormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios : GovukRadios,
+    input_radio: components_refactored.input_radio
```

Comment:
input_radio we can remove as can see we are using govukRadios only

Reply:
will resolve this with a push

Reply:
this is done now @jigneshkumarpatel